packages/@aws-cdk/aws-appmesh/lib/virtual-service.ts

```diff
@@ -27,6 +28,11 @@ export interface IVirtualService extends cdk.IResource {
    * The Mesh which the VirtualService belongs to
    */
   readonly mesh: IMesh;
+
+  /**
+   * Client policy for Virtual Service
+   */
+  readonly clientPolicy?: ClientPolicy;
```

Comment:
It's interesting to add this here. Should it also be added to `VirtualServiceAttributes`, so it can be set for imported `VirtualService`s too?